ci/actions/test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -ex
+
+cargo build --verbose
+cargo test --verbose -- --test-threads=1
+
+for example_dir in examples/*; do
+    if [ $example != 'examples/linalg' ]; then
```

Comment:
This package could also be tested, unless you are saying  there are failures there?

Reply:
I don't remember why I disabled this example, but anyway I need to update this example.
I'll enable it when I update it.

Reply:
I don't remember correctly, but maybe here `tox` is for avoiding `pip install -e .` in the local environment.

Reply:
Since `linalg` example is outdated, it's OK to skip it now. I'll update it later.

Reply:
Yeah, when I tested this on my branch the linalg example didn't compile: https://github.com/mtreinish/rust-numpy/runs/799258075?check_suite_focus=true#step:9:400 it's also skipped on travis:  https://github.com/PyO3/rust-numpy/blob/master/ci/travis/test.sh#L9 which is why I did it here